iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -173,14 +197,22 @@ inline void storable_function<Capacity, signature<ReturnType, Args...>>::copy(co
     auto ptr = dest.m_storage.template allocate<CallableType>();
     cxx::Expects(ptr != nullptr);
 
-    auto obj = static_cast<CallableType*>(src.m_callable);
+    // AXIVION Next Construct AutosarC++19_03-M5.2.8: type erasure - conversion to compatible type
+    const auto obj = static_cast<CallableType*>(src.m_callable);
+    cxx::Expects(obj != nullptr); // should not happen unless src is incorrectly used after move
+
+    // AXIVION Next Construct AutosarC++19_03-A5.3.2: obj is guaranteed not to be
+    // nullptr unless src was invalidated by move (illegal use of src)
```

Comment:
Please check if this suppression can be removed now that you add the check in line 202.

Reply:
According to local scan it is not. Removed.

Once we replace the always existing runtime check with an a compile time dependent assert it will probably be needed again (depending on the compile flag ...).